app/poi_api/poi.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from app import firestore_db, common
+from flask import jsonify
+
+
+class POI(common.FirebaseDataEntity):
+    def __init__(
+        self,
+        db_ref,
+        name,
+        clasification,
+        hours_of_operation,
+        address,
+        type,
+        longitude,
+        latitude,
+        image_url,
+    ):
+        super().__init__(db_ref)
+        if db_ref is None:
+            self.db_ref = firestore_db
+        self.name = name
+        self.classification = clasification
+        self.hours_of_operation = hours_of_operation
+        self.address = address
+        self.type = type
+        self.longitude = longitude
+        self.latitude = latitude
+        self.image_url = image_url
+
+    def get_all_POI():

```

Comment:
Don't feel like you have to encapsulate all the POI logic into this POI class. If you just want to make this a data class with smaller helper functions, while a service class handles the database calls that's fine. No changes needed here, but in the future this might grow _a lot_ due to all the functionality.